### PR TITLE
Add methods to comment in pull requests

### PR DESCRIPTION
**Pull request by nmatpt**

### CODE REVIEW

**Comment by rtfpessoa** [comment]
LGTM :thumbsup:

**Comment by mrfyda** [comment]
:+1:

File: src/main/scala/com/codacy/client/stash/Repository.scala [hunk 1]
```diff
@@ -4,7 +4,7 @@ import play.api.libs.functional.syntax._
 import play.api.libs.json._
 
 case class Repository(slug: String, name: String, scm: String, public: Boolean,
-                      cloneUrl: String, cloneUrls: Seq[RepositoryUrl])
+                      cloneUrls: Seq[RepositoryUrl])
```
Comment (pedrorijo91) [inline src/main/scala/com/codacy/client/stash/Repository.scala]: why this change?

Comment (nmatpt) [inline src/main/scala/com/codacy/client/stash/Repository.scala]: cloneUrl is not mandatory. In my project the parse failed because the field was not found

Comment (pedrorijo91) [inline src/main/scala/com/codacy/client/stash/Repository.scala]: not mandatory? but it's 0 or 1 right? wouldn't a option make more sense then? Or can be more than 1?
